test/e2e/specs/onboarding/setup__start-writing.ts

```diff
@@ -0,0 +1,154 @@
+				testUser.email,
+				testUser.username,
+				testUser.password
+			);
+		} );
+
+		it( "Start user's first post", async function () {
+			await page.waitForURL( /.*post-new.php\?start-writing=true.*/ );
+			editorPage = new EditorPage( page );
+		} );
+
+		it( 'Enter blog title', async function () {
+			await editorPage.enterTitle( postTitle );
+		} );
+
+		it( 'Publish post and open Launchpad', async function () {
+			await editorPage.publish();
+			await page.waitForURL( /.*start-writing\/launchpad.*/ );
+			await page.locator( ':text("Your blog\'s almost ready!")' ).waitFor();
+		} );
+
+		it( 'Add blog name and description', async function () {
+			await page.getByRole( 'button', { name: 'Name your blog' } ).click();
+			await page.getByPlaceholder( 'A catchy name to make your blog memorable' );
+			await page.fill(
+				'input[name="setup-form-input-name"]',
+				`Start writing site ${ testUser.username }`
+			);
+			await page.fill(
+				'textarea[name="setup-form-input-description"]',
+				`The place of ${ testUser.username }`
+			);
+			await page.click( 'button.setup-form__submit' );
+		} );
+
+		it( 'Navigate choose a domain', async function () {
+			await page.waitForURL( /.*start-writing\/launchpad.*/ );
+			await page.getByRole( 'button', { name: 'Choose a domain' } ).click();
+		} );
+
+		it( 'Search for a domain', async function () {
+			domainSearchComponent = new DomainSearchComponent( page );
+			await domainSearchComponent.search( testUser.username );
+		} );
+
+		it( 'Skip selecting a domain', async function () {
+			await Promise.all( [ page.waitForNavigation(), page.click( 'text=Decide later' ) ] );
+		} );
+
+		it( 'Navigate choose a plan', async function () {
+			await page.waitForURL( /.*start-writing\/launchpad.*/ );
+			await page.getByRole( 'button', { name: 'Choose a plan' } ).click();
+		} );
+
+		it( 'Select WordPress.com Personal plan', async function () {
+			await page.getByRole( 'button', { name: 'Get Personal' } ).click();
+		} );
+
+		it( 'Launch site', async function () {
+			await page.waitForURL( /.*start-writing\/launchpad.*/ );
+			await page.getByRole( 'button', { name: 'Launch your blog' } ).click();
+		} );
+
+		it( 'Land in checkout cart', async function () {
+			cartCheckoutPage = new CartCheckoutPage( page );
+			const totalAmount = await cartCheckoutPage.getCheckoutTotalAmount();
+			expect( totalAmount ).toBeGreaterThan( 0 );
+		} );
+
+		it( 'Enter billing and payment details', async function () {
+			const paymentDetails = DataHelper.getTestPaymentDetails();
+			await cartCheckoutPage.enterBillingDetails( paymentDetails );
+			await cartCheckoutPage.enterPaymentDetails( paymentDetails );
+		} );
+
+		it( 'Make purchase', async function () {
+			await cartCheckoutPage.purchase( { timeout: 90 * 1000 } );
+		} );
+
+		it( "Ensure we're redirected to celebration screen", async function () {
+			await page.waitForURL( /.*start-writing\/celebration-step.*/ );
+		} );
```

Comment:
**suggestion:non-blocking** - We've been having some intermittent issues where the plan purchase does not go through. No one seems to be able to help as to what is the cause, or whether it's happening on production as well. 

With that in mind, we do have other specs that test the simulated purchase flow. If purchasing a plan is not required (that is, the Free option suffices) then can we use that instead?

Reply:
Sounds good, [changed it here](https://github.com/Automattic/wp-calypso/pull/77457/commits/44bd69e347b45d53a5eca3d05d63bd602bb65860).